src/doc/unstable-book/src/library-features/asm.md

```diff
@@ -593,6 +595,7 @@ Here is the list of currently supported register classes:
 | wasm32 | `local` | None\* | `r` |
 | BPF | `reg` | `r[0-10]` | `r` |
 | BPF | `wreg` | `w[0-10]` | `w` |
+| AVR | `reg` | `r[0-31]`, `xh`, `xl`, `yh`, `yl`, `zh`, `zl` | `r` |
```

Comment:
`reg_xyz` is missing from the documentation.

Reply:
Will resolve once the various classes are ironed out.